content/java/concepts/array-list/terms/sort/sort.md

```diff
@@ -0,0 +1,106 @@
+---
+Title: '.sort()'
+Description: 'Used to sort arrays of primitive types and objects.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.sort()`** method is used to sort arrays of primitive types and objects. The sorting is done in ascending order by default, but it can be customized for objects by implementing the `Comparable` interface or using a `Comparator`.
+
+## Syntax
+
+Here's a basic syntax of how to use the `.sort()` method:
+
+```pseudo
+Arrays.sort(array);
```

Comment:
As you've added in the introduction of the method, there can be an optional parameter `Comparator`. Please add it to the syntax and write a one-line description below the syntax block.